examples/platform-app/package.json

```diff
@@ -13,10 +13,10 @@
         "format": "d2-style apply",
         "format:staged": "yarn format --staged",
         "cy:start": "BROWSER=none yarn start",
-        "cy:open": "concurrently 'yarn:cy:start' 'd2-utils-cypress open' --kill-others",
-        "cy:run": "concurrently 'yarn:cy:start' 'd2-utils-cypress run' --kill-others",
-        "cy:capture": "concurrently 'yarn:cy:start' 'd2-utils-cypress run --capture --serverMinorVersion 37' --kill-others",
-        "cy:stub": "concurrently 'yarn:cy:start' 'd2-utils-cypress run --stub --serverMinorVersion 37' --kill-others",
+        "cy:open": "wait-on 'http://localhost:3000' && cypress open",
+        "cy:run": "wait-on 'http://localhost:3000' && cypress run",
+        "cy:capture": "concurrently 'yarn cy:start' 'yarn cy:run --env network=capture' --kill-others --success first",
+        "cy:stub": "concurrently 'yarn cy:start' 'yarn cy:run --env network=stub' --kill-others --success first",
```

Comment:
`wait-on` is moved to the outside, and `concurrently` only exists on the outside (since the `npx` removal). It's a bit more verbose, but this can be extracted to a script in an app for clarity and brevity.

The `d2-utils-cypress install` command can be extended to set that up for us.